skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md

```diff
@@ -937,6 +938,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MeetingRecordingExpirationDays
+
+Specifies the number of days before meeting recordings will expire and move to the recycle bin. Value can be from 1 to 99,999 days.
```

Comment:
Nitpick: the value can be 1 to 99999 -- without the comma :-)

Reply:
Microsoft Writing Style Guide says: Use commas in numbers that have four or more digits.